tests/test_root.py

```diff
@@ -148,54 +151,12 @@ def test_root_gen_sign_verify():
     )
     rmd = signing.wrap_as_signable(rmd)
 
-    signed_portion = rmd["signed"]
+    # Sign it with the GPG key.
+    root_signing.sign_root_metadata_dict_via_gpg(rmd, SAMPLE_FINGERPRINT)
 
-    canonical_signed_portion = common.canonserialize(signed_portion)
-
-    if not SSLIB_AVAILABLE:
-        pytest.skip(
-            "--TEST SKIPPED⚠️ : Unable to perform GPG signing without "
-            "securesystemslib and GPG."
-        )
-        return
-
-    # gpg_key_obj = securesystemslib.gpg.functions.export_pubkey(
-    #         SAMPLE_FINGERPRINT)
-
-    gpg_sig = root_signing.sign_via_gpg(canonical_signed_portion, SAMPLE_FINGERPRINT)
-
-    signed_rmd = copy.deepcopy(rmd)
-
-    signed_rmd["signatures"][SAMPLE_KEYVAL] = gpg_sig
-
-    # # Dump working files
-    # with open('T_gpg_sig.json', 'wb') as fobj:
-    #     fobj.write(common.canonserialize(gpg_sig))
-
-    # with open('T_gpg_key_obj.json', 'wb') as fobj:
-    #     fobj.write(common.canonserialize(gpg_key_obj))
-
-    # with open('T_canonical_sigless_md.json', 'wb') as fobj:
-    #     fobj.write(canonical_signed_portion)
-
-    # with open('T_full_rmd.json', 'wb') as fobj:
-    #     fobj.write(common.canonserialize(signed_rmd))
-
-    # Verify using the SSL code and the expected pubkey object.
-    # # (Purely as a test -- we wouldn't normally do this.)
-    # verified = securesystemslib.gpg.functions.verify_signature(
-    #     gpg_sig, gpg_key_obj, canonical_signed_portion)
-
-    # assert verified
-
-    authentication.verify_gpg_signature(
-        gpg_sig, SAMPLE_KEYVAL, canonical_signed_portion
-    )
-
-    print(
-        "--TEST SUCCESS✅: GPG signing (using GPG and securesystemslib) and "
-        "GPG signature verification (using only cryptography)"
-    )
+    # Verify the signature.
+    root_signing.sign_via_gpg(rmd)
+    securesystemslib.gpg.functions.verify_signature(rmd)
```

Comment:
I think we want to use authentication.verify_gpg_signature, not securesystemslib.gpg.functions.verify_signature.

Reply:
I've reverted the test to (mostly) how it was